README.md

```diff
@@ -87,7 +93,7 @@ Useful NPM scripts:
 
 - **`npm test`** - builds and activates Karma test runner on PhantomJS.
 
-- **`npm run clean`** - blow out the `./dist` folder. 
+- **`npm run clean`** - blow out the `./dist` folder.
```

Comment:
just for clarity, maybe we could say "removes files from the `./dist` folder for a clean build"?

Reply:
definitely!